.gitattributes

```diff
@@ -0,0 +1 @@
+* text=auto eol=lf
```

Comment:
This repository is probably the only one where we test on Windows and are hitting these end-of-line issues. 

https://prettier.io/docs/en/options.html#end-of-line